src/backendMiddleware.ts

```diff
@@ -3,7 +3,7 @@ import { IdentityWallet } from 'jolocom-lib/js/identityWallet/identityWallet'
 import { EncryptionLib, EncryptionLibInterface } from 'src/lib/crypto'
 import { Storage } from 'src/lib/storage/storage'
 import { KeyChain, KeyChainInterface } from 'src/lib/keychain'
-import { ConnectionOptions } from 'typeorm/browser'
```

Comment:
This is very surprising!! You normally need to import from typeorm/browser for react-native. It doesn't really matter here since this just imports a type, but you've changed it everywhere I think yes? Does this really work as expected?? I have yet to test the branch on a phone